govc/vm/disk/change.go

```diff
@@ -73,6 +76,7 @@ func (cmd *change) Register(ctx context.Context, f *flag.FlagSet) {
 	f.IntVar(&cmd.key, "disk.key", 0, "Disk unique key")
 	f.StringVar(&cmd.mode, "mode", "", fmt.Sprintf("Disk mode (%s)", strings.Join(vdmTypes, "|")))
 	f.StringVar(&cmd.sharing, "sharing", "", fmt.Sprintf("Sharing (%s)", strings.Join(sharing, "|")))
+	f.Int64Var(&cmd.limit, "disk.io.limit", 0, "Disk storage IO per seconds limit (-1 for unlimited)")
```

Comment:
You can use a pointer instead, so the default would actually be `nil`, see for example: https://github.com/vmware/govmomi/blob/20fe92d2f45b3573da5fb830e397a52c5460484d/govc/flags/resource_allocation_info.go#L74